src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -144,10 +218,11 @@
           >
             <v-checkbox
               id="extrapro-registration-checkbox"
-              v-model="checkbox"
+              v-model="business.isConfirmed"
               class="pt-0 mt-0"
               hide-details
               :label="checkboxLabel"
+              :rules="getShowErrors ? [(v) => !!v] : []"
```

Comment:
This is this component:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/6a2563c6-c7c5-46f8-9a1e-c7fb2309c755)